inc/group.class.php

```diff
@@ -275,6 +275,7 @@ function showForm($ID, $options = []) {
 
       echo "<tr class='tab_bg_1'>";
       echo "<td class='subheader' colspan='2'>".__('Can contain');
+      echo "</td><td colspan='2' class='center'>";
```

Comment:
Previous row seems to contains only 2 `td`, so existing one using `colspan='2'` was already using the 2 columns.

Reply:
Previous rows are linked to rowspan='10' of comment field so we can update to rowspan='13' ?

Reply:
I guess all empty columns should be dropped, and rowspan adapted accordingly. By the way, it seems there are 12 rows total, not 13; and this one is the 10th; so @cedric-anne was right

Reply:
I do it